src/Router.tsx

```diff
@@ -30,23 +30,24 @@ export default function AppRouter(): JSX.Element {
   const [userId, setUserId] = useState<InsideToken>(null);
   const [userRole, setUserRole] = useState<InsideToken>(null);
 
-  const checkUserToken = () => {
+  const CheckUserToken = () => {
```

Comment:
Why Pascal case? This is a function, not a component

Reply:
ESLint ... do not know either